dev/bk-merge-pr.py

```diff
@@ -395,10 +397,32 @@ def get_reviewers(pr_num):
             if approval_msg in comment['body'].lower():
                 reviewers_ids.add(comment['user']['login'])
 
+    approval_review_states = ['approved']
+    pr_reviews = get_json('{0}/pulls/{1}/reviews'.format(GITHUB_API_BASE, pr_num), True)
+    for review in pr_reviews:
+        for approval_state in approval_review_states:
+            if approval_state in review['state'].lower():
+                reviewers_ids.add(review['user']['login'])
+
+    if len(reviewers_ids) == 0:
+        fail("No approvals found in this pull request")
+
     reviewers_emails = []
     for reviewer_id in reviewers_ids:
+        username = None
+        useremail = None
         user = get_json("%s/users/%s" % (GITHUB_API_URL, reviewer_id))
-        reviewers_emails += ['%s <%s>' % (user['name'].strip(), user['email'].strip())]
+        if user['email'] is not None:
+            useremail = user['email'].strip()
```

Comment:
Nit: you can replace the entire line if by

useremail = user['email'].strip() if user['email] or None